decidim-meetings/app/views/decidim/meetings/meetings/show.html.erb

```diff
@@ -0,0 +1,49 @@
+<% content_for(:title, translated_attribute(meeting.title)) %>
+
+<div class="row column view-header">
+  <h2 class="heading2"><%= translated_attribute meeting.title %></h2>
+  <div class="author-data">
+    <div class="author-data__main">
+      <div class="author author--inline">
+        <span class="author__avatar">
+          <%= image_tag meeting.author.avatar.url %>
+        </span>
+        <%= l meeting.created_at.to_date, format: :long %> <%= t(".by") %> <%= meeting.author.name %>
```

Comment:
@andreslucena "it's not necessary" does it mean that is optional or that we can get rid of it?